documentation/theme/using-component-themes.asciidoc

```diff
@@ -185,7 +185,7 @@ and those representations are added to the `theme` attribute values.
 If the component implements `HasTheme` interface (which is true for every component that has `addThemeVariants` method),
 then the following API can be used:
 
-.Adding theme values for the component that implements `HasTheme` interface
+.Adding theme values for the component that implements 'HasTheme' interface
```

Comment:
Did you change these intentionally?

Reply:
yes, backtics inside file name statement looks bad in  VC because they go to a code block instead to be inlined.